smartcontract/service/native/governance/param.go

```diff
@@ -0,0 +1,554 @@
+	if err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.ReadVarUint, deserialize peerPubkeyList length error!")
+	}
+	for i := 0; uint64(i) < n; i++ {
+		k, err := serialization.ReadString(r)
+		if err != nil {
+			return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.ReadString, deserialize peerPubkey error!")
+		}
+		this.PeerPubkeyList = append(this.PeerPubkeyList, k)
+	}
+
+	m, err := serialization.ReadVarUint(r, 0)
+	if err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.ReadVarUint, deserialize withdrawList length error!")
+	}
+	for i := 0; uint64(i) < m; i++ {
+		k, err := serialization.ReadUint32(r)
+		if err != nil {
+			return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.ReadString, deserialize withdraw error!")
+		}
+		this.WithdrawList = append(this.WithdrawList, k)
+	}
+	return nil
+}
+
+type Configuration struct {
+	N                    uint32
+	C                    uint32
+	K                    uint32
+	L                    uint32
+	BlockMsgDelay        uint32
+	HashMsgDelay         uint32
+	PeerHandshakeTimeout uint32
+	MaxBlockChangeView   uint32
+}
+
+func (this *Configuration) Serialize(w io.Writer) error {
+	if err := serialization.WriteUint32(w, this.N); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize n error!")
+	}
+	if err := serialization.WriteUint32(w, this.C); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize c error!")
+	}
+	if err := serialization.WriteUint32(w, this.K); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize k error!")
+	}
+	if err := serialization.WriteUint32(w, this.L); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize l error!")
+	}
+	if err := serialization.WriteUint32(w, this.BlockMsgDelay); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize block_msg_delay error!")
+	}
+	if err := serialization.WriteUint32(w, this.HashMsgDelay); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize hash_msg_delay error!")
+	}
+	if err := serialization.WriteUint32(w, this.PeerHandshakeTimeout); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize peer_handshake_timeout error!")
+	}
+	if err := serialization.WriteUint32(w, this.MaxBlockChangeView); err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.WriteUint32, serialize max_block_change_view error!")
+	}
+	return nil
+}
+
+func (this *Configuration) Deserialize(r io.Reader) error {
+	n, err := serialization.ReadUint32(r)
+	if err != nil {
+		return errors.NewDetailErr(err, errors.ErrNoCode, "serialization.ReadUint32, deserialize n error!")
+	}
+	this.N = n
```

Comment:
same comment as above, 
update this after all data members deserialized successfully.